baselines/asset/samples/generated/v1/asset_service.list_feeds.js.baseline

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+'use strict';
+
+function main() {
+  // [START asset_list_feeds_sample]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  /**
+   *  Required. The parent project/folder/organization whose feeds are to be
+   *  listed. It can only be using project/folder/organization number (such as
+   *  "folders/12345")", or a project ID (such as "projects/my-project-id").
+   */
+  // const parent = 'abc123'
+
+  // Imports the Asset library
+  const {AssetServiceClient} = require('@google-cloud/asset').v1;
+
+  // Instantiates a client
+  const assetClient = new AssetServiceClient();
+
+  async function listFeeds() {
+    // Construct request
+    const request = {
+      parent,
+    };
+
+    // Run request
+    const response = await assetClient.listFeeds(request);
```

Comment:
If a method like `listFeeds` has an equivalent `listFeedsAsync`, I would prefer that we only generate a sample for `listFeedsAsync`, and show the syntax:

```js
for async (const resp of ssetClient.listFeeds(request)) {
  console.info(resp);
}
```

I mention later on in review.

This could be follow on work IMO too, _I would open tracking issues on this repository for any improvements we want to make to sample generation._

Reply:
I actually think that's happening! When I generated this API IRL, I only got one sample for `listAssetsAsync`, even though it has a correlate `listAssets` function. I don't think `listFeeds` has an async version.

Reply:
Checked on https://github.com/googleapis/googleapis/blob/master/google/cloud/asset/v1/asset_service.proto
The `listFeed` is not following the [AIP-132](https://google.aip.dev/132). So it is unary call. That's why it doesn't have `Async` method.

Reply:
@sofisl shoed me that this was the case, retract my comment 😄